samples/csharp/MicrosoftEduGraphSamples/Program.cs

```diff
@@ -15,5 +15,6 @@
 // Create an instance of the class you want to test and call the desired workflow method.
 // Each flow represents a common entire process that can be tested using Microsoft Graph SDK, all the sample flows are located in the "workflows" folder.
 
-ModuleWorkflow workflow = new ModuleWorkflow(config);
```

Comment:
In Program.cs file, we usually keep the workflows related to the current PR workflows and delete the ones from Main. That is why Module workflow from previous PR changes are removed so that the current PR changes can be run by the user.

Customer can create objects for workflows and run the code.